<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>696</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Terminus</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>=ULTIMATE VOLAT TUNER=</string>
  </property>
  <property name="toolTip">
   <string comment="12223"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>366</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminus</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabAn">
    <attribute name="title">
     <string>An</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>0</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>c1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>c2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ch</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>0</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>raw</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>0</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>c3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>len</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>0</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>scal</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabRel_0_31">
    <attribute name="title">
     <string>Rel 0_31</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>5</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Terminus</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>00110100100001011101</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabRel_32_63">
    <attribute name="title">
     <string>Rel 32_63</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabTime">
    <attribute name="title">
     <string>Time</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabHelp">
    <attribute name="title">
     <string>Help</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Here will be help.
Next line.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonRefersh">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>640</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Send current active data."/>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatusBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>status line</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonTransmit">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>640</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Send current active data."/>
   </property>
   <property name="text">
    <string>Transmit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonReadRom">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>660</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Send current active data."/>
   </property>
   <property name="text">
    <string>Read ROM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonWriteRom">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>640</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Send current active data."/>
   </property>
   <property name="text">
    <string>Write ROM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLoadFile">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>660</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Send current active data."/>
   </property>
   <property name="text">
    <string>Load File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSaveFile">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>660</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Send current active data."/>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>600</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Сделать пиздато</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
